dart/lib/src/hub.dart

```diff
@@ -149,10 +128,10 @@ class Hub {
 
   /// Captures the message.
   Future<SentryId> captureMessage(
-    String message, {
-    SentryLevel level,
-    String template,
-    List<dynamic> params,
+    String? message, {
+    SentryLevel level = SentryLevel.info,
```

Comment:
```suggestion
    SentryLevel level,
```

Reply:
this is done o nthe SentryClient now

Reply:
I know but then SentryLevel is a required field (because it is not nullable) which would mean that anyone who is using the Hub directly would always need to specify a SentryLevel.

Reply:
we could simply `SentryLevel? level` then?

Reply:
Yes, so we would just keep the `Sentry` static API non-nullable and all the other nullable?

Reply:
SentryEvent.level itself is nullable, so all of them should be nullable, the difference is, captureMessage defaults to `info` on `SentryLevel` if none is set.

Reply:
Ah I see, I guess that's where most of my confusion comes from. Do all others default to `info`, too? Or just `captureMessage`?

Reply:
only `captureMessage`.
everything else is `error` by default, but nullable on our side, the backend infers that automatically if no level is available.

Reply:
got it

Reply:
there's still plenty of `SentryLevel? level = SentryLevel.info`
they all should be `SentryLevel? level` only, the reason is here:
https://github.com/getsentry/sentry-dart/blob/main/dart/lib/src/sentry_client.dart#L167

if we decide to change the level, we don't need to change in ~6 methods, but only one place.

Reply:
Yeah, I just pushed a commit which removes them.